docs/elements/overview.mdx

```diff
@@ -49,6 +49,8 @@ To get started, install the Clerk Elements package:
   ```
 </CodeBlockTabs>
 
+Also make sure that your [`moduleResolution`](https://www.typescriptlang.org/tsconfig/#moduleResolution) inside `tsconfig.json` is set to `bundler`. Otherwise you might run into issues with resolving TypeScript types from Clerk Elements.
```

Comment:
What do you think about a callout here?

```suggestion
<Callout type="info">
If your project uses TypeScript, make sure that [`moduleResolution`](https://www.typescriptlang.org/tsconfig/#moduleResolution) in`tsconfig.json` is set to `bundler`. Otherwise, you might run into issues with resolving TypeScript types from Clerk Elements.
</Callout>
```